.brv/context-tree/development/ito_workflow/pre_push_adversarial_code_review.abstract.md

```
@@ -0,0 +1 @@
+Ito agents should run an adversarial code review before pushing branches or opening PRs to catch low-hanging issues, block P0/P1 defects, and reduce review noise.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)